ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -84,21 +85,28 @@ export default function BuildQuote({
   const topAssets = useSelector(getTopAssets);
   const fromToken = useSelector(getFromToken);
   const toToken = useSelector(getToToken) || destinationTokenInfo;
-  const swapsEthToken = useSelector(getSwapsEthToken);
-  const fetchParamsFromToken =
-    sourceTokenInfo?.symbol === 'ETH' ? swapsEthToken : sourceTokenInfo;
+  const defaultSwapsToken = useSelector(getSwapsDefaultToken);
+  const chainId = useSelector(getCurrentChainId);
 
   const tokenConversionRates = useSelector(getTokenExchangeRates, isEqual);
   const conversionRate = useSelector(getConversionRate);
   const currentCurrency = useSelector(getCurrentCurrency);
 
+  const defaultTokenSymbol = SWAPS_CHAINID_DEFAULT_TOKEN_MAP[chainId].symbol;
```

Comment:
[Q] Do you expect defaultTokenSymbol to be different than `defaultSwapsToken.symbol` here? Seems like getSwapsDefaultToken returns this same object?

Reply:
No, I have addressed this as part of 04e7af609

Reply:
that commit is dependent on b20c98f2c